easybuild/easyconfigs/p/PGI/PGI-16.4-GCC-5.3.0-2.26.eb

```diff
@@ -0,0 +1,25 @@
+name = 'PGI'
+version = '16.4'
+
+homepage = 'http://www.pgroup.com/'
+description = "C, C++ and Fortran compilers from The Portland Group - PGI"
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['pgilinux-20%(version_major)s-%(version_major)s%(version_minor)s-x86_64.tar.gz']
+checksums = ['3804dc13b8cf3d50aa16fb56e2682dd7']
+
+gccver = '5.3.0'
+binutilsver = '2.26'
+versionsuffix = '-GCC-%s-%s' % (gccver, binutilsver)
+
+dependencies = [
+    ('GCCcore', gccver),
+    ('binutils', binutilsver, '', ('GCCcore', gccver)),
+    ('numactl', '2.0.11', '', ('GCCcore', gccver)),
+]
+
```

Comment:
@damianam please open an issue for dropping the `PGI_INSTALL_AMD` for newer versions (although it's not a big issue, if not happens because it's no longer supported, fine I guess)